Gruntfile.js

```diff
@@ -30,6 +30,7 @@ module.exports = function(grunt) {
     shell: {
       test: {
         command : '<%= config.binPath %>/istanbul cover --report lcov --dir test/reports/ <%= config.binPath %>/_mocha test/spec -- --reporter xunit-file',
+        //command : '<%= config.binPath %>/istanbul cover --report lcov --dir test/reports/ <%= config.binPath %>/_mocha test/spec -- --reporter spec',
```

Comment:
Yeah, it's on my todo list, just wasn't a focus right now. But I would appreciate your input.